src/Alert/index.jsx

```diff
@@ -74,6 +75,35 @@ const Alert = React.forwardRef(({
   );
 });
 
+// This is needed to display a default prop for Alert.Heading element
+// Copied from react-bootstrap since BaseAlert.Heading component doesn't have defaultProps,
+// so there seems to be no other way of providing correct default prop for base element
+const DivStyledAsH4 = divWithClassName('h4');
```

Comment:
I think this is a reasonable approach. Interesting `BaseAlert.Heading` doesn't have defaultProps specified despite defaultProps definitely existing lol. My one concern with this approach is dealing with future code drift between the props we specify here vs. what's in react-bootstrap but I think the value of having the props listed out in our Paragon docs is worth that risk.

When upgrading react-bootstrap, we'll just have to make sure to account for any changes in prop types for components like this to account for them on our end.

Reply:
Yeah, we probably need to make a list of components that use react-bootstrap, so we can quickly check everything once we upgrade.